docs/3.0/resources/upgrade-to-prefect-3.mdx

```diff
@@ -169,3 +169,66 @@ This change affects you if: You're using agents from an early version of Prefect
 </info>
 
 In Prefect 2, agents were deprecated in favor of next-generation workers. Workers are now standard in Prefect 3. For detailed information on upgrading from agents to workers, please refer to our [upgrade guide](https://docs-3.prefect.io/3.0/resources/upgrade-agents-to-workers).
+
+### Resolving common gotchas
+
+#### `AttributeError: Coroutine has no attribute <some attribute>`
```

Comment:
```suggestion
#### `AttributeError: coroutine has no attribute <some attribute>`
```

Reply:
was updated in https://github.com/PrefectHQ/prefect/pull/15271/commits/622fa16b63eab3b3fda732e423187ee973d3349f